### PR TITLE
Rename master to main

### PR DESCRIPTION
**Pull request by mfeurer**

### CODE REVIEW

**Review by PGijsbers — COMMENTED** [review]
According to PyCharm there are many references to `master` which are not updated. These are in markdown files (`contributing`, `pull request template` and `readme`),  `.rst` files (`usage`, `contributing`) and tutorials (e.g. `create_upload_tutorial.py`). Is that intentional?

**Comment by mfeurer** [comment]
> According to PyCharm there are many references to master which are not updated.

I had a file mask on...PyCharm only showed the files ending with `.py`...thanks for seeing that and sorry for the incomplete PR, I will update it.

**Comment by mfeurer** [comment]
Done. I had to replace a link referring to an anchor by a link to an example. I am currently wondering whether we should keep the deep link and ask at the sphinx gallery team how to do this?

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ It can be used to download or upload OpenML data such as datasets and machine le
 
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
-Master branch:
+Main branch:
 
 [![Build Status](https://travis-ci.org/openml/openml-python.svg?branch=master)](https://travis-ci.org/openml/openml-python)
```
Comment (PGijsbers) [inline README.md:18]: `build status` and `coverage` links should also be updated, right?

**Review by PGijsbers — CHANGES_REQUESTED** [review]
I tested the links and they all seem to work fine for me. I only have the concern for the badges otherwise it looks good to me.

File: README.md (hunk 1 above)
Comment (mfeurer) [reply README.md]: Yes and no. We don't have travis-ci any more, so I opted to remove all of them for now.

**Review by PGijsbers — APPROVED** [review]

**Pull request merged**